package.json

```diff
@@ -34,12 +34,12 @@
     "dependencies": {
         "@autorest/codemodel": "^4.15.0",
         "@autorest/extension-base": "^3.2.1",
-        "@azure-rest/core-client": "^1.0.0-beta.4",
+        "@azure-rest/core-client": "^1.0.0-beta.6",
+        "@azure-rest/core-client-paging": "^1.0.0-alpha.20210820.2",
         "@azure-tools/codegen": "^2.5.294",
         "@azure/core-client": "^1.1.2",
         "@azure/core-http": "^1.2.4",
         "@azure/core-lro": "^2.1.0",
-        "@azure/core-paging": "^1.1.3",
```

Comment:
is it for regular sdk paging ? Are we sure it's okay to remove it ?

Reply:
Good catch!

Reply:
do we import `@azure/core-paging` directly from code gen?

Reply:
Yeah for the High-level clients we import it. We just consume the PageableAsyncITerator type I think